config/services.yaml

```
@@ -0,0 +1,8 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    Torr\Storyblok\:
+        resource: ../src/*
+        exclude: ../src/{TorrStoryblokBundle.php}
```

I would prefer the array notation here:

```suggestion
        exclude:
             - ../src/TorrStoryblokBundle.php
```